base/managers.jl

```diff
@@ -153,18 +186,33 @@ let tunnel_port = 9201
 end
 
 
-# establish an SSH tunnel to a remote worker
-# returns P such that localhost:P connects to host:port
+"""
+    ssh_tunnel(user, host, bind_addr, port, sshflags) -> localport
+
+Establish an SSH tunnel to a remote worker.
+Returns a port number `localport` such that `localhost:localport` connects to `host:port`.
+"""
 function ssh_tunnel(user, host, bind_addr, port, sshflags)
-    localp = next_tunnel_port()
-    ntries = cnt = 100
-    while !success(detach(`ssh -T -a -x -o ExitOnForwardFailure=yes -f $sshflags $(user)@$host -L $localp:$bind_addr:$(Int(port)) sleep 60`)) && cnt > 0
-        localp = next_tunnel_port()
+    port = Int(port)
+    cnt  = 100
+    localport = next_tunnel_port()
+    # if we cannot do port forwarding, bail immediately
+    # the connection is forwarded to `port` on the remote server over the local port `localp`
+    # the -f option backgrounds the ssh session
+    # `sleep 60` command specifies that an alloted time of 60 seconds is allowed to start the
+    # remote julia process and estabilish the network connections specified the the process topology.
+    # If no connections are made within 60 seconds, ssh will exit and an error will be printed on the
+    # process that launched the remote process.
+    ssh = `ssh -T -a -x -o ExitOnForwardFailure=yes`
+    while !success(detach(`$ssh -f $sshflags $user@$host -L $localp:$bind_addr:$port sleep 60`)) && cnt > 0
```

Comment:
Whoops, thanks.  #13696 is not caused by this as that report was from yesterday.